tests/unit/apps.bats

```diff
@@ -26,3 +26,12 @@ load test_helper
   echo "status: "$status
   assert_success
 }
+
+@test "(apps) apps:rename" {
+  deploy_app
+  run bash -c "dokku apps:rename $TEST_APP great-test-name"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+  run bash -c "dokku apps:destroy great-test-name"
```

Comment:
you'll want to pass in `--force`. it's hanging on confirmation of `apps:destroy`